content/en/docs/how-tos/cluster-claim.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Creating a Cluster Pool"
+description: How to create a cluster pool from which a job can claim a cluster.
+---
+
+DPTP integrates Hive into the OpenShift test platform to manage the cluster pools and clusters in them. There is a defined pool [ci-cluster-pool](https://github.com/openshift/release/tree/master/clusters/app.ci/ci-cluster-pool) from which can be used to claim an OpenShift cluster for a test. In this page, we document how to create a new cluster pool with the cloud account provided by a CI user.
```

Comment:
IMO we should really be more user oriented here: what problem does this doc solve. The first paragraph should re-assure the user they came to the right place. The user does not care at all about DPTP or HIve. The user is very likely an "owner" or a lead of some group or team, they have resources at their disposal (cloud accounts), and they want to set things so that their jobs can use these accounts. The paragraph should be about that.